src/components/ProjectList/ProjectItem.js

```diff
@@ -6,7 +6,7 @@ import Col from 'react-bootstrap/lib/Col'
 import Row from 'react-bootstrap/lib/Row'
```

Comment:
Sure, I expected you are going to rename them soon :) but they are still there ...